Sources/StreamChat/Audio/AudioRecorder/AudioRecording.swift

```diff
@@ -37,7 +37,7 @@ public protocol AudioRecording {
 // MARK: - Implementation
 
 /// Definition of a class to handle audio recording
-open class StreamAudioRecorder: NSObject, AudioRecording, AVAudioRecorderDelegate {
+open class StreamAudioRecorder: NSObject, AudioRecording, AVAudioRecorderDelegate, AppStateObserverDelegate {
```

Comment:
The AudioRecorder now will observe AppState changes and when the app moves to the background will stop recording rather than waiting for the system to kill it.